roofit/roofitcore/inc/RooBinWidthFunction.h

```diff
@@ -68,6 +68,8 @@ class RooBinWidthFunction : public RooAbsReal {
     return _histFunc->plotSamplingHint(obs, xlo, xhi);
   }
 
+  bool divideByBinWidth() const { return _divideByBinWidth; }
+  const RooHistFunc* histFunc() const { return &(*_histFunc); }  
```

Comment:
```suggestion
  const RooHistFunc& histFunc() const { return *_histFunc; }
```
Hist func is never a `nullptr`, so you can return a `const` reference.